map_server:
  ros__parameters:
    map_topic: map_server/map
    map_frame: map

# List of sensors
    sensors: [ sensor0, sensor1 ]


    sensor0:
        topic: vl53l1x/range
        device:
            type: vl53l1x
            addr: 0x29
            timeout: 60000
        transform:
            frame: vl53l1x
            posX: 0.03
            posY: -0.07
            posZ: 0.05
            roll: 0.0
            pitch: -1.57
            yaw: 0.0
    sensor1:
        topic: sensor_two/range
        device:
            type: SRF10
            addr: 0xE2
            timeout: 60000
        transform:
            frame: sensor1
            posX: 0.08
            posY: -0.05
            posZ: 0.05
            roll: 0.0
            pitch: 0.0
            yaw: 0.0


